sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/EventingTestKit.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.testkit;
+
+import akka.annotation.InternalApi;
+import com.google.protobuf.ByteString;
+import kalix.javasdk.Metadata;
+
+import java.util.Collection;
+
+@InternalApi
+public interface EventingTestKit {
+
+  String getHost();
+  Integer getPort();
+
+  Topic getTopic(String topic);
+
+  interface Topic {
+    Message<ByteString> expectNext();
+
+    Collection<Message<ByteString>> expectAll();
```

Comment:
It will be nice if we have the means to deserialize it automatically, something like: `Foo foo = expectNext(Foo.class)`

If we want to take that path, we might prefer to call this method: `expectNextMessage
 and `expectAllMessages` to signal that is the one returning the raw message.

Reply:
We have typed methods now in https://github.com/lightbend/kalix-jvm-sdk/pull/1586/commits/16657c9337192685494b4052ba5e3bc55e35b1a5